frontends/web/.eslintrc.js

```diff
@@ -126,8 +126,6 @@ module.exports = {
     "prefer-rest-params": 2,
     "prefer-spread": 2,
     "prefer-template": 0, // we can enable this when they add an option to allow concat for complex variables
-    // "quote-props": [2, "as-needed"],
-    "quote-props": "off",
```

Comment:
The actual rule was added here https://github.com/digitalbitbox/bitbox-wallet-app/pull/1012/files#diff-2d0c462789f03f7ffd5d19adeba7dbe4R154